src/python/test/test_dxclient.py

```diff
@@ -2781,36 +2782,90 @@ def _get_analysis_id(dx_run_output):
         # don't actually need it to run
         run("dx terminate " + watched_job_id)
 
-        # --ssh implies --priority high
+        # --ssh implies high priority when --priority is not specified
```

Comment:
https://jenkins-vstg.internal.dnanexus.com/job/dx-toolkit/job/dxpy-branch-integration-tests-on-staging-20.04-python3/60/console
@commandlinegirl If starting from --ssh test the `dx_run_output` is not defined. I guess even Jenkins could not launch the ssh test as I mentioned :( 
Previously `dx_run_out` here was not from this job but the `watched_job`.
======================================================================
ERROR: test_dx_run_priority (test_dxclient.TestDXClientRun)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/dx-toolkit/src/python/test/dxpy_testutil.py", line 694, in wrapper
    retval = func(*args, **kwargs)
  File "/dx-toolkit/src/python/test/test_dxclient.py", line 2792, in test_dx_run_priority
    ssh_job_id = dx_run_output.split('\n')[0]
UnboundLocalError: local variable 'dx_run_output' referenced before assignment

----------------------------------------------------------------------
Ran 1 test in 2.054s

FAILED (errors=1)
$ dx new project other --brief
project-G666x3j0Kk70zjVZ30vvZ0Xz

$ dx cd project-G666x3j0gYgF43gf24jp1k27:/

$ dx run myapplet -y --ssh --brief